src/utils/utils.ts

```diff
@@ -96,7 +96,7 @@ export function baseUrlFromContext(mattermostSiteUrl: string): string {
 export function makeBulletedList(pretext: string, items: string[]): string {
     let text = '* ' + items.join('\n* ');
     if (pretext) {
-        text = `###  ${pretext}\n` + text;
+        text = `####  ${pretext}\n` + text;
```

Comment:
Not sure if this is suggestion is easier to read. Just seems weird that we're using backtick syntax and "plus" string concatenation.

```suggestion
        text = `####  ${pretext}\n${text}`
```

Reply:
done.